src/models/CommonSchema.ts

```diff
@@ -7,7 +7,8 @@ export class CommonSchema<T> {
     enum?: any[];
     items?: T | T[];
     properties?: { [key: string]: T; };
-    additionalProperties?: boolean | T;
+    additionalProperties?: T;
```

Comment:
`additionalProperties` can be also false/true. Why you changed it?

Reply:
Cause it was a mistake. For the `CommonModel` it should not be possible to be a boolean, it should be a `CommonModel`. For `Schema` it can be `Schema | Boolean` which it still can because of the generic type.